File: java8/One.java
import java.io.*;
import java.lang.Exception;

public class One {

    public static void main(String[] args) {
        System.out.println("What is your name?");
        String name = new String();
        try {
            BufferedReader br = new BufferedReader(new
                    InputStreamReader (System.in));
            name = (String) br.readLine();
        } catch (Exception e) {
            e.printStackTrace();
        }
        System.out.println("Hello, " + name + ", nice to meet you!");
    }

}

File: csharp/One/One.cs
using System;

namespace ConsoleApplication
{
    public class One
    {
        public static void Main(string[] args)
        {

		 Console.WriteLine("What is your name? ");
		 string person = Console.ReadLine();
		 Console.WriteLine("Hello, {0}, nice to meet you!", person);       
        }
    }
}
